ksql-cli/src/test/java/io/confluent/ksql/cli/CliTest.java

```diff
@@ -86,10 +94,15 @@
 import org.junit.experimental.categories.Category;
 import org.junit.rules.RuleChain;
 import org.junit.rules.Timeout;
+import org.junit.runner.RunWith;
+import org.mockito.InOrder;
+import org.mockito.Mock;
+import org.mockito.junit.MockitoJUnitRunner;
 
 /**
  * Most tests in CliTest are end-to-end integration tests, so it may expect a long running time.
  */
+@RunWith(MockitoJUnitRunner.class)
```

Comment:
It seems a little weird that this test has both integration tests and unit tests in it. (I switched the existing unit tests over to Mockito and also added more.) Would it make more sense to have separate files for the CLI integration and CLI units tests?

Reply:
Yes!